.github/workflows/ci.yml

```diff
@@ -77,12 +78,16 @@ jobs:
             - name: Set output variables
               id: bake_metadata
               env:
-                  BAKE_METADATA: ${{ steps.bake.outputs.metadata }}
+                  BAKE_TEMPLATE_META: ${{ steps.meta.outputs.bake-file }}
               run: |
-                  targets=$(echo "${BAKE_METADATA}"| jq -c 'keys')
-                  echo "::set-output name=targets::${targets}"
-                  images=$(echo "${BAKE_METADATA}"| jq -c '. as $base |[to_entries[] |{"key": (.key|ascii_upcase|sub("-"; "_"; "g") + "_IMAGE"), "value": [(.value."image.name"|split(",")[0]),.value."containerimage.digest"]|join("@")}] |from_entries')
-                  echo "::set-output name=images::${images}"
+                  cat ${BAKE_TEMPLATE_META} | jq -c \
+                    | .github/workflows/merge-bake-template-target.sh \
```

Comment:
It looks like these lines are the same as on lines 66-67 in "Generate docker-bake meta file".
So instead you could reuse it and read the `docker-bake-template-meta.json` file directly here to avoid duplication?

Reply:
Changed and tests passed as usual :wink: